cirq-core/cirq/transformers/transformer_api.py

```diff
@@ -274,14 +293,13 @@ def transformer(cls_or_func: Any) -> Any:
         method = cls.__call__
 
         @functools.wraps(method)
-        def method_with_logging(
-            self, circuit: 'cirq.AbstractCircuit', context: TransformerContext
-        ) -> 'cirq.AbstractCircuit':
+        def method_with_logging(self, circuit, **kwargs) -> 'cirq.AbstractCircuit':
             return _transform_and_log(
-                lambda circuit, context: method(self, circuit, context),
+                lambda circuit, **kwargs: method(self, circuit, **kwargs),
                 cls.__name__,
                 circuit,
-                context,
+                _get_context(method, f'{cls.__name__}.__call__()', **kwargs),
```

Comment:
Looks like the second `name` arg to `_get_context` is never used.

Reply:
removed.